packages/at_secondary_server/test/sync_unit_test.dart

```diff
@@ -469,6 +469,39 @@ void main() {
         expect(syncResponse[3]['operation'], '*');
       });
 
```

Comment:
Please add another test which both uses a regex and sets skipDeletes: true

Reply:
Done